packages/core/src/renderbuffer/RenderbufferSystem.js

```diff
@@ -0,0 +1,203 @@
+import { Renderbuffer } from './Renderbuffer';
+import { System } from '../System';
+
+/**
+ * System plugin to the renderer to manage renderbuffers.
+ *
+ * @memberof PIXI.systems
+ */
+export class RenderbufferSystem extends System
+{
+    constructor(renderer)
+    {
+        super(renderer);
+
+        /**
+         * List of renderbuffers that have allocated RBOs so they can be disposed
+         * in the future.
+         *
+         * @member {Array<PIXI.Renderbuffer>}
+         * @private
+         */
+        this.managedRenderbuffers = [];
+
+        /**
+         * Renderbuffer value that tells us we don't know what is bound.
+         *
+         * @member {PIXI.Renderbuffer}
+         */
+        this.unknownRenderbuffer = new Renderbuffer(10, 10);
```

Comment:
Not a biggee, but do we need this? Could we not always have the main screen bound by default?

Reply:
The unknown renderbuffer means you don’t know what Renderbuffer is attached - if it is the screen, you have to put null. When we initialize the context, I’m assuming it as unknown.

This would also be useful when accessing the renderbuffer directly in webgl and then telling the system that it should set to unknown.